### PR TITLE
Enable CONFIG_ARM_SMMU and CONFIG_ARM_SMMU_V3 in aarch64.

### PR DESCRIPTION
**Pull request by ddaney-fungible**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

Enable IOMMU in aarch64 kernel. It is needed to allow VFIO to be used with Mariner.

###### Change Log  

- Enable CONFIG_ARM_SMMU and CONFIG_ARM_SMMU_V3 in aarch64 kernel.

###### Does this affect the toolchain?  

NO

###### Associated issues  

- #6818

###### Test Methodology

- local build

### CODE REVIEW

**Comment by charles-gray** [comment]
Nice!!

**Comment by christopherco** [comment]
Hi @ddaney-fungible thanks for the contribution! I will request to retarget your PR to our `main` branch instead of the `2.0` branch.
`main` is our 2.0 development branch so changes will flow there first.

File: SPECS/kernel/kernel.spec [hunk 1]
```diff
@@ -28,7 +28,7 @@
 Summary:        Linux Kernel
 Name:           kernel
 Version:        5.15.137.1
-Release:        1%{?dist}
+Release:        2%{?dist}
```
Comment (christopherco) [inline SPECS/kernel/kernel.spec:31]: Please add a corresponding changelog entry in the `%changelog` section for this `-2` release increment

File: SPECS/kernel-headers/kernel-headers.spec [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Summary:        Linux API header files
 Name:           kernel-headers
 Version:        5.15.137.1
-Release:        1%{?dist}
+Release:        2%{?dist}
```
Comment (christopherco) [inline SPECS/kernel-headers/kernel-headers.spec:4]: Please add a corresponding changelog entry in the `%changelog` section for this `-2` release increment

File: SPECS-SIGNED/kernel-signed/kernel-signed.spec [hunk 3]
```diff
@@ -10,7 +10,7 @@
 Summary:        Signed Linux Kernel for %{buildarch} systems
 Name:           kernel-signed-%{buildarch}
 Version:        5.15.137.1
-Release:        1%{?dist}
+Release:        2%{?dist}
```
Comment (christopherco) [inline SPECS-SIGNED/kernel-signed/kernel-signed.spec:13]: Please add a corresponding changelog entry in the `%changelog` section for this `-2` release increment

**Review by christopherco — COMMENTED** [review]
Added a couple other comments, but aside from retargeting to `main`, the changes seem reasonable to me. Once retargeted, @rlmenge let's kick off our official buddy builds to test fully out this change

We'll also want to make sure this change goes into our `3.0-dev` branch so it is part of our upcoming major 3.0 release.

**Comment by ddaney-fungible** [comment]
Thanks for the feedback, I don't know how to change the branch target of a PR, so I am going to close this one and open a new one for the main branch.

**Pull request closed without merging**